scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/LiveJobs.java

```diff
@@ -691,14 +692,14 @@ TerminationData killTask(JobId jobId, String taskName) throws UnknownJobExceptio
             if (taskData == null) {
                 throw new IllegalStateException("No information for: " + task.getId());
             }
-
-            TerminationData terminationData = TerminationData.newTerminationData();
-            terminationData.addTaskData(taskData, false);
-
             TaskResultImpl taskResult = new TaskResultImpl(task.getId(),
                     new TaskAbortedException("Aborted by user"), new SimpleTaskLogs("", "Aborted by user"),
                     System.currentTimeMillis() - task.getStartTime());
 
+            TerminationData terminationData = TerminationData.newTerminationData();
+            terminationData.addTaskData(jobData.job, taskData, false, taskResult);
```

Comment:
This 2 lines of code seems repeated 3 times. Could you wrap these instructions in a private method with a sensible name?